docs/components/badge.md

```diff
@@ -13,24 +13,39 @@ something.
 
 ### appearance
 
-The Button component can take the following appearance values:
+The Badge component can take the following appearance values:
 
 - `success`
-- `danger`
 - `warning`
+- `danger`
+- `info`
 - `default`
 - `custom`
 
-<KBadge appearance="success">LOW</KBadge>
-<KBadge appearance="warning">MEDIUM</KBadge>
-<KBadge appearance="danger">HIGH</KBadge>
-<KBadge>DEFAULT BADGE</KBadge>
+<KBadge appearance="success">SUCCESS</KBadge>
+<KBadge appearance="warning">WARNING</KBadge>
+<KBadge appearance="danger">DANGER</KBadge>
+<KBadge appearance="info">INFO</KBadge>
+<KBadge>DEFAULT</KBadge>
+
+```vue
+<KBadge appearance="success">SUCCESS</KBadge>
+<KBadge appearance="warning">WARNING</KBadge>
+<KBadge appearance="danger">DANGER</KBadge>
+<KBadge appearance="info">INFO</KBadge>
+<KBadge>DEFAULT</KBadge>
+```
+
+### shape
+
+The Badge has two shapes that can be changed with a `shape` property
+
+<KBadge appearance="warning">Round</KBadge>
+<KBadge appearance="warning" shape="rectangular">Rectangular</KBadge>
 
 ```vue
-<KBadge appearance="success">HIGH</KBadge>
-<KBadge appearance="warning">MEDIUM</KBadge>
-<KBadge appearance="danger">LOW</KBadge>
-<KBadge>DEFAULT BADGE</KBadge>
+<KBadge appearance="warning">Round</KBadge>
+<KBadge appearance="warning" shape="rectangle">Rectangular</KBadge>
```

Comment:
```suggestion
<KBadge appearance="warning" shape="rectangular">Rectangular</KBadge>
```

Reply:
Fixed and also resolved the comment with a Vue warning as well.